### PR TITLE
Create an easy way to prepare and shutdown server locally

### PR DESCRIPTION
**Pull request by mslovik**
Add prepare and shutdown scripts

### CODE REVIEW

File: prepare.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Locate server directory
+SERVER_DIR="$(jar tf ${SERVER_ZIP} | head -1 | cut -d '/' -f 1)"
+echo "Server directory: $SERVER_DIR"
+
+# Set JBOSS_HOME
+export JBOSS_HOME=$SERVER_DIR
+
+if [ "$MODE" == "domain" ]; then
+    $SERVER_DIR/bin/domain.sh &
+    $SERVER_DIR/bin/jboss-cli.sh -c '/host=master/core-service=management/management-interface=http-interface
```
Comment (hpehl) [inline prepare.sh:23]: `/host=master` should be replaced with `/host=primary`.

**Review by hpehl — CHANGES_REQUESTED** [review]
`/host=master` should not work with WildFly > 27 (see also https://issues.redhat.com/browse/WFLY-16149)

File: prepare.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+if [ "$MODE" == "domain" ]; then
+    $SERVER_DIR/bin/domain.sh &
+    $SERVER_DIR/bin/jboss-cli.sh -c '/host=primary/core-service=management/management-interface=http-interface
+                                      :undefine-attribute(name=security-realm)
+                                      :reload'
+    sleep 40
+elif [ "$MODE" == "domain-hc-dc" ]; then
+    # Setup dc, please setup hc manually
+    cp -r $SERVER_DIR/domain/ $SERVER_DIR/dc/
+    $SERVER_DIR/bin/domain.sh --host-config=host-master.xml -Djboss.domain.base.dir=$SERVER_DIR/dc &
```
Comment (hpehl) [inline prepare.sh:30]: Please replace _all_ occurrences of `master/slave` with `primary/secondary`. Thanks!

**Review by hpehl — CHANGES_REQUESTED** [review]

File: prepare.sh (hunk 2 above)
Comment (mslovik) [reply prepare.sh]: Sorry for the insufficient fix. I already added a conditional check that sets the host based on the configuration located in a server directory.

**Comment by hpehl** [comment]
LGTM! Thanks @mslovik

**Review by hpehl — APPROVED** [review]

**Pull request merged**